content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+to the email.
+
+Here are the volunteer positions currently available.
+
+### Photographer
+
+* Each event will have a main photographer and a back up photographer. In case
+the main photographer has a last minute emergency, they should contact the
+committee immediately so that the backup can be informed.
+* **You will need:** A camera of your own or one that you can borrow.
+* People who are not willing to be photographed will be in lanyards so make sure that you don’t photograph them!
+* Photographs should be sent to the committee as soon as possible after
+ the event is over.
+* **Commitment:** Full length of the event (< 3 hours) Since
+this position is on an event-by-event basis, you’ll have to fully attend the
+event that you’re photographing.
+
+### Poster Posters
+
+* 1-2 per event.
+* Notice will be given 4-5 days before the event.  You will have
+to coordinate with the director of publicity to pick up printed posters,
+supplies for putting them up, and deciding on locations to put them up.
+* **Low commitment:** 1-1.5 hours/event, can do them at your own pace.
+
+### Mentor for Workshops or Technical Events
+
+* Depending on the size of the event, could range from 3-7 per event.  Notice
+for this will be given 14-10 days before the event, with an explanation of the
+content being covered and the difficulty level.
+* **You will need:**
+ - Enthusiasm for the topic of the workshop!
+ - Enough expertise in the material being presented or taught to help the
+   attendees of the workshop/event if they’re stuck or have questions.
+* **Commitment**: This is a higher commitment position than the previous
+positions. In addition to attending the event, you will also have to spend
+time familiarizing yourself with the material prior to it.
+
+### WiCS-Sys Volunteer
+
+* This is a great way to learn about Linux systems administration, version
+control, and static website development!
+* You’ll work with our WiCS systems committee and help maintain our machines, track issues, etc.
+* Indicate your interest in being a wics-sys volunteer in the form above. Doing so will add
+* you to the wics-sys mailing list.
+* **You will need:** to be willing to work independently, try new things, follow along with existing work, and ask
+questions. Prior experience, although appreciated, is not a pre-requisite.
+
+### Run an event with us!
+
+* **You** have to reach out to us [by email](mailto:wics-ugrad@lists.uwaterloo.ca) with an event that you’d like to
+see or run on your own!
+* You’ll play a major role in running and organizing the event.
+* We’ll help you advertise it, book rooms for it, order food for it under our budget.
+* **Commitment:** 5-6 hours spread out over around 10 days.
+* Examples of such events can be:
+ - Let us know about a potential speaker, get our approval, and bring them to give a talk.
+ - Let us know about a workshop that you'd like to run, get our approval, and run it.
```

Comment:
the approval stuff could go in the original bullet points (before the examples) since it applies to everything

you could also keep it as a verb and say things "Bring in a speaker" "Run a workshop" - not sure if this is better than Fatema's suggestion, but throwing it out there